common.mk

```diff
@@ -0,0 +1,5 @@
+SHELL=/bin/bash
+
+ifeq ($(findstring python3.6, $(shell ls -ls /usr/bin/python* | grep python3.6 2>&1)),)
+$(error Please install Python 3.6 on the system)
```

Comment:
What is `error`? I don't have that on my Mac. Is it a Linux command?

I would think you would to exit with a non-zero exit code here, although maybe error already does that.

Reply:
`error` is a make keyword. If the equivalence is false it return to stdout with that error message.

Reply:
Got it; my bad.